src/framework/components/anim/component.js

```diff
@@ -130,6 +149,11 @@ class AnimComponent extends Component {
         this._speed = value;
     }
 
+    /**
+     * @name AnimComponent#activate
+     * @type {boolean}
+     * @description If set to true, the anim component will automatically begin playing when all animations have been assigned to its state graph.
```

Comment:
the description sounds more like for a setter, not getter. Should be 'if returns true ...' instead of 'if set to true' ..' ?

Reply:
This property isn't read only like some of the others.

Reply:
ah I see .. the diff didn't show the setter only getter .. all good!